buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijPlugin.kt

```diff
@@ -42,36 +42,22 @@ fun Project.configureIntellijPlugin() {
   configure<ChangelogPluginExtension> {
     val PROSPECTIVE_RELEASE_VERSION: String by extra
     version.set("v$PROSPECTIVE_RELEASE_VERSION")
-    unreleasedTerm.set("v$PROSPECTIVE_RELEASE_VERSION")
     headerParserRegex.set(Regex("""v\d+\.\d+\.\d+"""))
     path.set("${project.projectDir}/CHANGE-NOTES.md")
   }
 
   val changelog = extensions.getByType(ChangelogPluginExtension::class.java)
 
-  val verifyVersionTask = tasks.register("verifyChangeLogVersion") {
```

Comment:
Hmm and if version.gradle.kts and latest entry in CHANGE-NOTES.md point to different versions... is that gonna be detected in any way?

Reply:
the same way I guess - I didn't change that

Reply:
Hmmm so far it was covered by `verifyChangeLogVersion` IIUC

Reply:
...or wasn't it? 🤔

Reply:
not really - it's unhandled exception but rather readable 
<img width="1252" alt="image" src="https://user-images.githubusercontent.com/19799111/199115414-e3834097-4022-438a-983b-ee18ba057b14.png">

Reply:
eh, it's a bit messy - version.gradle.kts vs latest entry in CHANGE-NOTES.md is not directly checked anywhere

Reply:
ok, this should work correctly now